### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-7: ordinal not in range(128)

### ISSUE THREAD

**Issue by DKorytkin** [DKorytkin/json_checker#24]
```python
    Checker(expected).validate(current)
/usr/local/lib/python2.7/dist-packages/checker.py:331: in validate
    result = checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:279: in validate
    return dict_checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:155: in validate
    result = checker.validate(current_value)
/usr/local/lib/python2.7/dist-packages/checker.py:272: in validate
    return list_checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:103: in validate
    result = checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:279: in validate
    return dict_checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:155: in validate
    result = checker.validate(current_value)
/usr/local/lib/python2.7/dist-packages/checker.py:279: in validate
    return dict_checker.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:155: in validate
    result = checker.validate(current_value)
/usr/local/lib/python2.7/dist-packages/checker.py:281: in validate
    return self.expected_data.validate(data)
/usr/local/lib/python2.7/dist-packages/checker.py:224: in validate
    result = checker.validate(current_data)
/usr/local/lib/python2.7/dist-packages/checker.py:302: in validate
    return _format_error_message(self.expected_data, data)
/usr/local/lib/python2.7/dist-packages/checker.py:69: in _format_error_message
    _format_data(current_data),
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

data = 'Самсонов'

    def _format_data(data):
        if callable(data):
            return data.__name__
        if data is None:
            return data
        if type(data) is str:
            return json.dumps(data)
>       return str(data)
E       UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-7: ordinal not in range(128)

/usr/local/lib/python2.7/dist-packages/checker.py:64: UnicodeEncodeError

```

**Comment by DKorytkin** [comment]
fixed chande to repr()
need fix
```python
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/IPython/core/interactiveshell.py", line 2882, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-3-e91f278ed96c>", line 1, in <module>
    Checker(str).validate(u'вывпывп')
  File "/home/dkorytkin/json_checker/checker.py", line 325, in validate
    result = checker.validate(data)
  File "/home/dkorytkin/json_checker/checker.py", line 279, in validate
    result = type_checker.validate(data)
  File "/home/dkorytkin/json_checker/checker.py", line 115, in validate
    raise TypeCheckerError(self._format_errors())
TypeCheckerError: 
current value u'\u0432\u044b\u0432\u043f\u044b\u0432\u043f' is not str

```

**Comment by DKorytkin** [comment]
```python
self = <json_checker.TypeChecker object at 0x7f911045f850>, current_data = 'Заканчивается'

    def validate(self, current_data):
>       log.info('Run item validation {}'.format(current_data))
E       UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-12: ordinal not in range(128)

/usr/local/lib/python2.7/dist-packages/json_checker.py:134: UnicodeEncodeError

```

**Comment by DKorytkin** [comment]
must be fixed in #33
